### PR TITLE
Settings for Publisher ID

### PR DESCRIPTION
**Pull request by whyisjake**
Add a settings panel for the publisher ID

### CODE REVIEW

File: simple-facebook-instant-articles.php [hunk 1]
```diff
@@ -74,6 +78,34 @@ public function __construct( $file, $version ) {
...
+		$this->options       = get_option( 'fb_instant' );
+	}
+
+	/**
+	 * The register activation hook should flush the rewrite rules, in the event
+	 * that it doesn't let's go ahead and flush the rules.
+	 *
+	 * @return void
+	 */
+	public function flush_rules() {
```
Comment (dashaluna) [inline simple-facebook-instant-articles.php:90]: @whyisjake what's the use case for this, why do we need to do this?

Comment (whyisjake) [inline simple-facebook-instant-articles.php:90]: There has been a lot of feedback feeds don't work when a person activates the plugin. This ensures the manual step of updating permalinks. Is there a conflict on VIP or elsewhere?

Comment (dashaluna) [inline simple-facebook-instant-articles.php:90]: Not sure, was just curios. Will try to dig into it at some point. I wonder if it's due to the fact that a plugin is added not via WP admin, for example as must-use or as a submodule?

Comment (dashaluna) [inline simple-facebook-instant-articles.php:90]: I've asked around and was told that flushing isn't working in core particularly well: https://core.trac.wordpress.org/ticket/30501

It needs to be done on activating/deactivating the plugin, plugin updates that change the registered rules, stuff like that.

**Pull request merged**